src/utils/settingsValidation/splitFilters.ts

```diff
@@ -187,3 +187,24 @@ export function validateSplitFilters(log: ILogger, maybeSplitFilters: any, mode:
 
   return res;
 }
+
+export function flagSetsAreValid(log: ILogger, method: string, flagSets: string[], flagSetsInConfig: string[]): string[] {
+  let toReturn: string[] = [];
+  if (flagSets.length === 0) {
```

Comment:
I think you can remove lines 193 to 196, because `validateSplits` validates for empty string too.

Also remove line 192 and update line 198 as follows: `let toReturn = ...`